tests/runfiles/linux.run

```diff
@@ -408,7 +408,8 @@ tests = ['zdb_001_neg', 'zfs_001_neg', 'zfs_allow_001_neg',
     'zpool_offline_001_neg', 'zpool_online_001_neg', 'zpool_remove_001_neg',
     'zpool_replace_001_neg', 'zpool_scrub_001_neg', 'zpool_set_001_neg',
     'zpool_status_001_neg', 'zpool_upgrade_001_neg', 'arcstat_001_pos',
-    'arc_summary_001_pos', 'arc_summary_002_neg', 'dbufstat_001_pos']
+    'arc_summary_001_pos', 'arc_summary_002_neg', 
+    'arc_summary3_001.pos', 'arc_summary3_002.pos', 'dbufstat_001_pos']
```

Comment:
The test case is named `arc_summary3_001_pos` not `arc_summary3_001.pos`.  The `.ksh` will be added to the test framework if needed.  Plus these is no `arc_summary3_002.pos` test case in this PR.  This is why you're getting the verification failures.